### PR TITLE
Create puerto-rico.nrel-op.json

### PR DESCRIPTION
**Pull request by sebastianbarry**
Created initial config file with details filled in from the MOU

### CODE REVIEW

**Comment by sebastianbarry** [comment]
No `mode_studied` in the MOU, so I left the field blank

File: configs/puerto-rico.nrel-op.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://puerto-rico-openpath.nrel.gov/api/",
```
Comment (sebastianbarry) [inline configs/puerto-rico.nrel-op.json:5]: @shankari can you please confirm which `connectUrl` we want for Puerto Rico? Thanks!

Comment (shankari) [reply configs/puerto-rico.nrel-op.json:5]: the base URL should be https://uprm-nicr-openpath.nrel.gov

Comment (sebastianbarry) [inline configs/puerto-rico.nrel-op.json:5]:
```suggestion
        "connectUrl": "https://uprm-nicr-openpath.nrel.gov/api/",
```

Comment (sebastianbarry) [reply configs/puerto-rico.nrel-op.json]: fixed

**Comment by shankari** [comment]
@sebastianbarry as I indicated in several of the other PRs, you need to rename the file as well.
You changed the code to pull the config from github 😄 - you must remember that we read the config file using the deployment name

**Comment by sebastianbarry** [comment]
> @sebastianbarry as I indicated in several of the other PRs, you need to rename the file as well. You changed the code to pull the config from github 😄 - you must remember that we read the config file using the deployment name

Resolved in https://github.com/e-mission/nrel-openpath-deploy-configs/pull/16/commits/265f47e5ae62756b35069c652532efde9ee50975

**Review by shankari — APPROVED** [review]

**Pull request merged**